GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -168,7 +168,7 @@ GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage() {
     resultData = [NSKeyedArchiver archivedDataWithRootObject:obj
                                        requiringSecureCoding:YES
                                                        error:error];
-    if (*error) {
+    if (resultData == nil && *error) {
```

Comment:
Isn't resultData == nil **OR** *error an encoding failure?

Reply:
they should coincide, but the real problem was an occasional deference of a NULL pointer. Really, I should be `resultData == nil || (checking error != NULL && *error != nil)`